doc/usecases/elektra_web/UC_setup_instance.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Setup instance
+
+## Summary
+
+Title: Setup instance
+Scope: Setup
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User sets up a daemon on the instance and connects it to elektra-web.
+
+## Scenarios
+
+Precondition: Setup
+Main success scenario: User successfully sets up elektrad on the instance (by
+	installing the dependencies and starting elektrad) and connects it to
+	elektra-web (clusterd) by specifying the host and an API key in a config file.
```

Comment:
@markus2330 I have already implemented cluster configuration in the backend, just decided to remove it in the frontend to keep the focus on the editor. I would keep the name because it still provides that functionality, but if you have a better idea I can also rename it :smile: